web/blob/templates/_base.html

```diff
@@ -38,8 +38,8 @@
             <li><a href="{{$consoles}}">Consoles</a></li>
             {{ end }}
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
-            <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
-            <li><a href="{{ pathPrefix }}/">Status</a></li>
+            <li><a href="{{ pathPrefix }}/">Graph</a></li>
```

Comment:
I think the graph page should still be `/graph`, and the root page just redirect to whatever we deem sensible to be the first thing to see.